types/signatures.go

```diff
@@ -96,17 +96,17 @@ type (
 	// must have a height >= 'Timelock'.
 	//
 	// 'PublicKeys' specifies the set of keys that can be used to satisfy the
-	// UnlockConditions; of these, at least 'NumSignatures' unique keys must sign
+	// UnlockConditions; of these, at least 'RequiredSignatures' unique keys must sign
 	// the transaction. The keys that do not need to use the same cryptographic
 	// algorithm.
 	//
-	// If 'NumSignatures' == 0, the UnlockConditions are effectively "anyone can
-	// unlock." If 'NumSignatures' > len('PublicKeys'), then the UnlockConditions
+	// If 'RequiredSignatures' == 0, the UnlockConditions are effectively "anyone can
+	// unlock." If 'RequiredSignatures' > len('PublicKeys'), then the UnlockConditions
 	// cannot be fulfilled under any circumstances.
 	UnlockConditions struct {
-		Timelock      BlockHeight
-		PublicKeys    []SiaPublicKey
-		NumSignatures uint64
+		Timelock           BlockHeight
+		PublicKeys         []SiaPublicKey
+		RequiredSignatures uint64
```

Comment:
NumSignatures is confusing because multisig has 2 "num signatures". There's the number of signatures required, and there's the number of different keys that you can use.

hmm. I guess that's not so confusing. Maybe it was okay the other way.